packages/ui/src/components/IconButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { SizeTokens } from 'tamagui';
+
+import { Button } from '../components/Button';
+
+export function IconButton({
+  children,
+  onPress,
+  size = '$s',
+}: {
+  children: React.ReactNode;
+  onPress: () => void;
+  size?: SizeTokens;
+}) {
+  return (
+    <Button size={size} onPress={onPress} borderWidth="unset">
```

Comment:
Why `borderWidth="unset"`?

Reply:
Because Button has a border and if we leave it on there the icon buttons look pretty strange: 
![image](https://github.com/tloncorp/tlon-apps/assets/1221094/4305359f-3670-4661-8c12-210dd9cab4a4)

Reply:
Makes sense, could you add a comment?

Reply:
Sure thing.